lib/cusolver/dense_generic.jl

```diff
@@ -122,7 +122,7 @@ function Xgeqrf!(A::StridedCuMatrix{T}) where {T <: BlasFloat}
 end
 
 # Xsytrs
-function sytrs!(uplo::Char, A::StridedCuMatrix{T}, p::CuVector{Int64}, B::StridedCuMatrix{T}) where {T <: BlasFloat}
+function sytrs!(uplo::Char, A::StridedCuMatrix{T}, p::CuVector{Int64}, B::StridedCuVecOrMat{T}) where {T <: BlasFloat}
```

Comment:
@kshyatt You probably need to adapt the computation of `nrhs` if `B` is a `StridedCuVector`.
A test will be also great to ensure that we didn't missed something else.

Reply:
Sorry, can you clarify what test you'd like a little? (I haven't had coffee yet). We have a test for `CuVector` `B` in:
`@test Array(Symmetric(d_A) \ d_b) ≈ (Af \ bf)` down below, no?

Reply:
OK, test added!